test/common/testlib.py

```diff
@@ -93,6 +93,21 @@
 opts.jobs = 1
 opts.fetch = True
 
+default_layouts = [
```

Comment:
Let's document this format a bit, and also mention test/browser-layouts.json here.

Reply:
Yes, that should hopefully answer your questions as well.